planex/mock.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+"""
+planex-mock: Wrapper around mock
+"""
+
+import subprocess
+import sys
+from uuid import uuid4
+
+import argparse
+import argcomplete
+from planex.util import add_common_parser_options
+
+
+def parse_args_or_exit(argv):
+    """
+    Parse command line options
+    """
+    parser = argparse.ArgumentParser(
+        description='Planex build system in a chroot (a mock wrapper)')
+    add_common_parser_options(parser)
+    parser.add_argument(
+        "--configdir", metavar="CONFIGDIR", default=None,
+        help="Change where the config files are found")
+    parser.add_argument(
+        "--resultdir", metavar="RESULTDIR", default=None,
+        help="Path for resulting files to be put")
+    parser.add_argument(
+        "-D", "--define", default=[], action="append",
+        help="--define='MACRO EXPR' \
+              define MACRO with value EXPR for the build")
+    argcomplete.autocomplete(parser)
+    return parser.parse_known_args(argv)
+
+
+def extract_srpms(passthrouhg_args):
+    """
+    Filter the list of passthrouhg arguments to check if they are all
```

Comment:
Typo: `pass-trough` (hyphen and two letters transposed). I'd also say below "a list of `these` SRPMs..".